src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -22,16 +27,39 @@
  * Represents the in-memory model of the address book data.
  */
 public class ModelManager implements Model {
+    public static final String MESSAGE_NO_NEXT_SESSION_FOUND = "Don't worry, no next session for you this week!";
```

Comment:
Question, would this be just no sessions in general? Like "no sessions found/added"? i feel that saying this week implies the following week might have a session, which wouldnt make sense considering they add according to day and not date. So the only time there would be no next session would be if there were no sessions at all

Reply:
errr cus we assuming the sessions repeat every week right? so if today is monday at 8am, assuming there are sessions in the week like wed 08:00, thur 08:00 then it will show wed 08:00 but if like today is friday 8am, then the sessions for the week is finished but next week, the sessions will repeat right?

Reply:
To elaborate: this message is shown when the system time is after all the sessions in the pupilist session datas (for the week since the sessions repeat every week)

Reply:
Hmm I agree with Christel as well. I understand where you are coming from but how about you change it to something like no more session til this Sunday (Assuming Sunday is the last day of the week)

Reply:
yeah the sessions repeat weekly but are you saying that if its sunday and the next session is monday ure not gonna show "next session: monday" ? like they have to wait for the next week to find out when the next session is? my understanding of the next up feature was just gonna be showing whatever the next one was, which is what i meant in the first comment. but then if you show it weekly there's the issue of what i said here. imo just showing the next session would suffice even if its sunday and the next one is on monday cos its not like they can skip weeks so its either every week has a session or none at all

Reply:
ohhh okk i gets sry my idea is to show what they hv in the week then when theyre done w the week they dont need to worry until the next week HAHHAHA

ok will issue a refactor for that spec instead